src/api/UsersRepository.js

```diff
@@ -25,6 +25,10 @@ export default {
         return kcRequest().then(axiosInstance => axiosInstance.get(`${resource}/${userId}/${clientRoleMappings}/${clientId}/available`));
     },
 
+    getUserActiveClientRoles(userId, clientId) {
```

Comment:
I'm slow to add documentation, but I think we should add some to help developers understand the difference between all the different role APIs. I think it makes sense to add them in the Repositories because they're like a service classes and will change less often that the UserInfo.vue component.